cordova-lib/src/plugman/registry/registry.js

```diff
@@ -218,10 +205,18 @@ module.exports = {
 
 /**
  * @method initSettings
+ * @param {Boolean} using npm registry
  * @return {Promise.<Object>} Promised settings.
  */
-function initSettings() {
+function initSettings(npm) {
     var settings = module.exports.settings;
+    var registryURL = 'http://registry.cordova.io';
+
+    //if npm is true, use npm registry
+    if(npm) {
+        registryURL = 'http://registry.npmjs.org';
+    }
+
```

Comment:
TBH, I prefer `if(npm) {...}` to `?`. I think it is just easier to read and leaves us open to add other changes if we realize we need to. If I am unaware of some performance boost or something, please let me know.